### ISSUE TITLE
Grid should support WBS column like Gantt

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#6697]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=43&t=24818&p=123281#p123281)

Hi,

Can I control the rownumber column like attachment?

Or use another column for row number?

Thank you.
![1.png](https://forum.bryntum.com/download/file.php?id=22106)

**Comment by juanprog97** [comment]
Hello, have you never made any progress on this requirement?

**Comment by matsbryntse** [comment]
No, but this is something you could implement quite easily on your own. Look at the Core/data/Wbs class to help you.

**Comment by juanprog97** [comment]
"Yes, it's true, there are different ways to edit that 'row counter' column.
